op-node/rollup/superchain.go

```diff
@@ -94,12 +84,15 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		PlasmaConfig:           plasma,
 	}
 
+	if chConfig.ChannelTimeoutBedrock != nil {
```

Comment:
do we need to return an error if  `ChannelTimeoutBedrock` is missing in the config? I gather that `ChannelTimeoutGranite` is optional at this point?

Reply:
In my eyes, we have a separation of duties. The `LoadOPStackRollupConfig` function reads from the superchain module and converts the `superchain.ChainConfig` struct into a `rollup.Config` struct. Then we have a `Config.Check` function that checks the validity of the `Config`. That is where we return errors, including one for `ErrMissingChannelTimeout` [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-node/rollup/types.go#L272)